src/utils/encoreLogOutTimer.js

```diff
@@ -55,16 +59,40 @@ function EncoreLogOutTimer() {
     let timeTillLogOut = (time > 0) ? time : 0;
 
     // Only for testing. If the function is run for tests, set the timeout no longer than 2 seconds
-    if (isTest && timeTillLogOut > 2000) {
-      timeTillLogOut = 2000;
+    if (isTest && timeTillLogOut > 8000) {
+      timeTillLogOut = 8000;
     }
 
     setTimeout(() => {
       utils.deleteCookie('PAT_LOGGED_IN');
       utils.deleteCookie('ENCORE_LAST_VISITED');
       utils.deleteCookie('nyplIdentityPatron');
+      this.loadLogoutIframe(isTest);
     }, timeTillLogOut);
   };
+
+  /**
+   * loadLogoutIframe(isTest)
+   * The function that loads a temporary iframe with the log out endpoint
+   * to completely log out the user from Encore. It then deletes the iframe right away.
+   * The reason to use this way to load the endpoint is to bypass the CORS loading from the client
+   * since III does not want to provide us a real log out API URI.
+   * * @param {isTest} - If running this method for testing, delete the iframe right away
+   */
+  this.loadLogoutIframe = (isTest) => {
+    const logoutIframe = document.createElement('iframe');
+    const [body] = document.getElementsByTagName('body');
+    const iframeExistingTime = isTest ? 100 : 10000;
```

Comment:
Adjust the time for the iframe to last for 10 seconds. If the codes are run for testing, it will only last for 1/10 seconds.